modules/fedimint-wallet-common/src/config.rs

```diff
@@ -61,6 +61,8 @@ pub struct WalletConfig {
 pub struct WalletConfigLocal {
     /// Configures which bitcoin RPC to use
     pub bitcoin_rpc: BitcoinRpcConfig,
+    /// Our own peer_id
+    pub our_peer_id: PeerId,
```

Comment:
I don't know if we should put it here, or is there's a better way.

Reply:
Maybe this should be a parameter of `ServerModuleInit::init` instead?

https://github.com/fedimint/fedimint/blob/6c15b0ea133b5239085884c14c334623287d555b/fedimint-core/src/module/mod.rs#L583-L588

Reply:
Maybe. Would be easier if Re https://github.com/fedimint/fedimint/issues/3025 was ready. :D

Reply:
Why not both? :shrug: :D